target/linux/ipq40xx/image/Makefile

```diff
@@ -198,6 +198,21 @@ define Device/cilab_meshpoint-one
 endef
 TARGET_DEVICES += cilab_meshpoint-one
 
+define Device/xiaomi_meshd01
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := Xiaomi
+	DEVICE_MODEL := Mesh D01
```

Comment:
This should correspond to the model string in DTS.

Despite, what exactly is the D01? If it is some kind of hardware revision, it might be more appropriate to use
`DEVICE_VARIANT := D01`

Reply:
it's not a revision i think, more like internal model name
miwifi_d01_firmware_52359_1.0.68.bin - is a stock fw file name
same with other xiaomi routers, for example router3 pro: miwifi_r3p_firmware_f5d80_2.16.29.bin

Reply:
I see. So, not DEVICE_VARIANT needed.